L1Trigger/L1THGCal/python/hgcalBackEndLayer2Producer_cfi.py

```diff
@@ -177,6 +177,12 @@
     InputCluster = cms.InputTag('hgcalBackEndStage1Producer:HGCalBackendStage1Processor'),
     ProcessorParameters = be_proc.clone()
     )
+    
+hgcalBackEndStage2SAtruncationProducer = cms.EDProducer(
+    "HGCalBackendLayer2Producer",
+    InputCluster = cms.InputTag('hgcalBackEndStage1ProducerSA:HGCalBackendStage1ProcessorSA'),
```

Comment:
Shouldn't need the `SA` in the names?

Reply:
This whole producer definition should have been removed in my previous commit. I removed it now.